packages/components/Callout/macos/FRNCallout.h

```diff
@@ -0,0 +1,12 @@
+//
+//  FRNCallout.h
+//  Pods
+//
+//  Created by Patrick Boyd on 9/24/24.
+//
+
+#ifndef FRNCallout_h
+#define FRNCallout_h
+
+
+#endif /* FRNCallout_h */
```

Comment:
Maybe just add a comment this is needed for cocoa pods to allow you to use the `FRNCallout-Swift.h` header?

Reply:
```suggestion
//
//  FRNCallout.h
//  Pods
//
//  Created by Patrick Boyd on 9/24/24.
//
// Created as a dummy file to enable importing FRNCallout-Swift.h and invoking Swift functions from Obj-C
//

#ifndef FRNCallout_h
#define FRNCallout_h


#endif /* FRNCallout_h */
```